snapcraft/tests/unit/project_loader/grammar/test_processor.py

```diff
@@ -57,55 +58,55 @@ class BasicGrammarTestCase(GrammarTestCase):
                 'foo',
                 'bar',
             ],
-            'target_arch': 'amd64',
+            'host_arch': 'x86_64',
             'expected_packages': {'foo', 'bar'}
         }),
         ('mixed including', {
             'grammar': [
                 'foo',
-                {'on i386': ['bar']}
+                {'on amd64': ['bar']}
             ],
-            'target_arch': 'i386',
+            'host_arch': 'x86_64',
             'expected_packages': {'foo', 'bar'}
         }),
         ('mixed excluding', {
             'grammar': [
                 'foo',
                 {'on i386': ['bar']}
             ],
-            'target_arch': 'amd64',
+            'host_arch': 'x86_64',
             'expected_packages': {'foo'}
         }),
         ('on amd64', {
             'grammar': [
                 {'on amd64': ['foo']},
                 {'on i386': ['bar']},
             ],
-            'target_arch': 'amd64',
+            'host_arch': 'x86_64',
             'expected_packages': {'foo'}
         }),
         ('on i386', {
             'grammar': [
-                {'on amd64': ['foo']},
-                {'on i386': ['bar']},
+                {'on i386': ['foo']},
+                {'on amd64': ['bar']},
```

Comment:
I really don't understand why you felt it necessary to switch these all around :frowning_face: .